# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/afbinit/hotfix-gcc5.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./afbinit.c.vanilla	2022-07-18 16:32:13.740250839 +0200
+++ ./afbinit.c	2022-07-18 16:32:23.704250441 +0200
@@ -7,6 +7,8 @@
  * Copyright (C) 1999 David S. Miller (davem@redhat.com)
  */
 
+#include <stdlib.h>
+#include <string.h>
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
